FirebaseFirestore.podspec

```diff
@@ -100,7 +100,9 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
   s.dependency 'abseil/time', abseil_version
   s.dependency 'abseil/types', abseil_version
 
-  s.dependency 'gRPC-C++', '~> 1.44.0'
+  s.dependency 'gRPC', '~> 1.57.0'
```

Comment:
FYI I think you'll need to upgrade grpc to at least 1.59.0 in order to pick up the iOS 12 crash fix (https://github.com/grpc/grpc/pull/34416). 1.59.0 is in "prerelease" at the time of writing. Presumably, it will be officially released in the next few weeks.

Reply:
Yeah, this is only to test if we can upgrade at all. Will do 1.59 later.